src/script/entity/message/ContentMessage.ts

```diff
@@ -51,6 +51,7 @@ export class ContentMessage extends Message {
   public readonly other_likes: ko.PureComputed<User[]>;
   public readonly failedToSend: ko.Observable<{queued?: QualifiedUserClients; failed?: QualifiedId[]} | undefined> =
     ko.observable();
+  public readonly storedBlob: ko.Observable<Blob | undefined> = ko.observable();
```

Comment:
```suggestion
  /** raw content of a file that was supposed to be sent but failed. Is undefined if the message has been successfully sent  */
  public readonly fileData: ko.Observable<Blob | undefined> = ko.observable();
```

Reply:
`Blob` is not really a good name. It's as if you named a field `boolean`, is doesn't hold any application meaning :)